hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/artifacts/upload/FileTransferHandlerVaadinUpload.java

```diff
@@ -147,14 +148,15 @@ public OutputStream receiveUpload(final String fileName, final String mimeType)
      */
     @Override
     public void updateProgress(final long readBytes, final long contentLength) {
-        if (readBytes > maxSize || contentLength > maxSize) {
-            LOG.error("User tried to upload more than was allowed ({}).", maxSize);
-            interruptUploadDueToFileSizeExceeded(maxSize);
```

Comment:
check if method still used

Reply:
was used in FileTransferHandlerStreamVariable; will use same approach there too; removed interruptUploadDueToFileSizeExceeded message